packages/dashboard/src/components/Tile/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useUniqueIdentifier } from '@rainbow-modules/hooks';
+import RenderIf from 'react-rainbow-components/components/RenderIf';
+import { colorToRgba, replaceAlpha } from 'react-rainbow-components/styles/helpers/color';
+import isChecked from './helpers/isChecked';
+import Checkmark from './checkmark';
+import {
+    StyledContainer,
+    StyledInput,
+    StyledLabelInput,
+    StyledLabel,
+    StyledValue,
+    StyledCheckmarkContainer,
+} from './styled';
+
+export default function Tile(props) {
+    const {
+        label,
+        value: valueProp,
+        color,
+        backgroundColor,
+        name: nameProp,
+        id,
+        className,
+        style,
+    } = props;
+    const { ariaDescribedby, groupName, onChange, value, multiple } = {};
+
+    const inputId = useUniqueIdentifier('tile-input');
+
+    const type = multiple ? 'checkbox' : 'radio';
+    const name = groupName || nameProp;
+    const checked = isChecked(multiple, value, nameProp);
+
+    const labelStyle = { color: replaceAlpha(colorToRgba(color), 0.7) };
+
+    return (
+        <StyledContainer id={id} className={className} style={style}>
+            <StyledInput
```

Comment:
I think we should not use an input when the component is presentational, it only should be used as an input with `focus` and `selected` styles when it is used inside the `TilePicker` component

Reply:
@maxxgreene @LeandroTorresSicilia I think we should discuss that